caculator.js

```diff
@@ -1,33 +1,25 @@
-function add_number() {
-    var value1=parseInt(document.getElementById("number").value);
-    var value2=parseInt(document.getElementById("numbertwo").value);
-    var sum=value1+value2;
-    document.getElementById("result").value=sum;
-    alert(sum);
+function calculate(meth)
+{
+var a=Number(document.cal.num1.value);
```

Comment:
here you should be getting comfortable with getElementById and parseInt rather then these classes and methods you have yet to be taught.